paradox-nakadi-consumer-boot/src/main/java/de/zalando/paradox/nakadi/consumer/boot/ControllerConfiguration.java

```diff
@@ -147,5 +155,19 @@ public EventHandlerController(final EventReceiverRegistry registry, final Client
                     elem.getEventName().equals(eventName)
                         && (null == consumerName || elem.getConsumerName().equals(consumerName));
         }
+
+        private DeferredResult<ResponseEntity<?>> getBadDeferredResult() {
+            final DeferredResult<ResponseEntity<?>> deferredResult = new DeferredResult<>();
+            deferredResult.setErrorResult(badRequest().body("Consumer name does not match with event type."));
```

Comment:
Message should be something like `Consumer not found.`. We are not matching the consumerName with the eventType.